src/SourceBuild/content/build.sh

```diff
@@ -180,7 +182,15 @@ while [[ $# > 0 ]]; do
     -use-mono-runtime)
       properties="$properties /p:SourceBuildUseMonoRuntime=true"
       ;;
-
+    -testnobuild)
+      test_no_build=true
+      ;;
+    -test-projects)
+      # In order to support multiple values delimited by semicolons, it needs to be properly quoted so that
+      # MSBuild can parse it correctly. See https://github.com/dotnet/sdk/issues/8792#issuecomment-393756980
+      properties="$properties /p:DotNetTestProjectsToRun='\"$2\"'"
+      shift
+      ;;
```

Comment:
I would rather use the official Arcade extension point which is already well known by repo contributors. Is that possible? https://github.com/dotnet/arcade/blob/0ffb8e9f4a064d8c77f07420dc399baec9daef89/eng/common/build.ps1#L68

Reply:
The `Projects` / `-projects` property/switch signals which projects to pass in to the Build.proj / build wrapper. The action that is performed on that project depends on the action being passed in, i.e. `-test`, `-restore`, `-build`, etc. This switch would make it easier to build individual task projects or build and test test projects inside the VMR.

I don't think this is beyond the scope of this PR as I disagree with the `-test-projects` switch. It should just be `-projects` to be consistent with all other Arcade-ified repos.

Reply:
Ok, it'll be consistent in name then but not in functionality (because it won't apply to build), so how does that help?

Reply:
I've reverted the changes to support this: https://github.com/dotnet/installer/pull/19222/commits/28ddf277572e32ae696a73f2dd4c2d38b714040f